packages/adminrouter/extra/src/test-harness/tests/test_metrics.py

```diff
@@ -35,6 +34,46 @@ def test_metrics_prometheus(self, master_ar_process):
         assert resp.headers['Content-Type'] == 'text/plain'
         assert resp.text.startswith('# HELP nginx_vts_info Nginx info')
 
+    def test_metrics_prometheus_static_upstreams_annotated(self, master_ar_process):
+        """
+        /nginx/metrics returns metrics in Prometheus format that are properly
+        annotated for static upstreams
+        """
+        static_upstream_annotations = {
+            '/acs/api/v1': 'IAM',
+            '/system/health/v1': 'DCOSDiagnostics',
+            '/system/checks/v1': 'DCOSChecks',
+            '/navstar/lashup/key': 'DCOSNet',
+            '/net/': 'DCOSNet',
+            '/pkgpanda': 'Pkgpanda',
+            '/system/v1/logs/': 'Log',
+            '/system/v1/metrics/': 'DCOSMetrics',
+            'exhibitor': 'Exhibitor',
```

Comment:
Exhibitor is missing a slash here I suppose.

Reply:
It should probably make a request to `/exhibitor/exhibitor/v1/cluster/status` instead.